models/response.go

```diff
@@ -1,5 +1,10 @@
 package models
 
+type ResponseHeader struct {
```

Comment:
Could we be using the [builtin Header type](https://golang.org/pkg/net/http/#Header) here, exported from net/http?

Reply:
`myHeader := http.Header{"Content-Type": {"Foo"}, "Requires-Authentication": {"bar", "buzz"}}`

Reply:
nice catch, refactored the code to use http.Header instead of custom ResponseHeader in commit 10525b5